README.md

```diff
@@ -39,12 +39,15 @@ vars:
     youtube_analytics_database: your_database_name 
 ```
 ### YouTube Video Metadata
-The Fivetran YouTube Analytics connector currently does not support video metadata. As such, it may be difficult to derive analysis from individual videos without understanding which video belongs to which record. 
 
-To alleviate this pain point, we have provided a solution which will allow you to create a [Fivetran Functions Connector](https://fivetran.com/docs/functions) that will sync your YouTube video metadata into a table in your warehouse. This dbt package can then use the `videos` metadata table to enrich your Fivetran YouTube Analytics reporting data. For more on how to create the Functions Connector, you can refer to our [YouTube Analytics Video Metadata Cloud Function](to_be_created_write-up_link_here) write up which provides detailed steps and code on how to configure the function. 
+The Fivetran YouTube Analytics connector currently does not support video metadata. Consequently, it may be difficult to analyze individual video data without knowing which video belongs to which record. 
+
+As a workaround, you can create a [Functions connector](https://fivetran.com/docs/functions) that syncs your YouTube video metadata into a table in your destination. This dbt package can then use the `VIDEOS` metadata table to enrich your YouTube Analytics reporting data. To learn more about creating a Functions connector, read our [YouTube Analytics Video Metadata Cloud Function article](to_be_created_write-up_link_here). It provides code and detailed steps on how to configure the function. 
```

Comment:
Please update this link when the one-pager is ready

Reply:
Will do, should be done by EOW!